maven-plugin/src/it/plugin-test/pom.xml

```diff
@@ -113,6 +113,13 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.googlecode.maven-java-formatter-plugin</groupId>
+        <artifactId>maven-java-formatter-plugin</artifactId>
+        <configuration>
+          <configFile>${project.basedir}/../../../../contrib/Eclipse-Accumulo-Codestyle.xml</configFile>
```

Comment:
Might also be good to include some additional configuration in the pluginManagement section in the parent POM, to enforce LF line endings.